stdlib.sh

```diff
@@ -136,7 +150,11 @@ EOC
 # Externally set control-variables:
 #
 # STDLIB_WANT_MEMCACHED	- Load native memcached functions;
-# STDLIB_API		- Specify the stdlib API to adhere to.
+# STDLIB_API		- Specify the stdlib API to adhere to;
+# STDLIB_COLOUR_MAP - Specify custom colour map file, default being
+#                     '/usr/local/lib/stdlib.colours';
+# STDLIB_WANT_COLOURISATION - Enables/disables colourised output, using values
+#                             defined in '/usr/local/lib/stdlib.colours'.
```

Comment:
I need to state the default API version, it would also be helpful to specify the default state of `STDLIB_WANT_COLOURISATION`.